app.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express')
+const app = express()
+
+//---Controller Functions---
+//Topics controllers
+const { getTopics } = require("./controllers/topics.controller.js")
+
+//--Endpoints--
+app.get("/api/topics", getTopics)
+
+
+//--Errors--
+app.all("*", (req, res, next) => {
+    res.status(404).send({ msg: `${req.originalUrl} is an invalid endpoint` }
+    )
+})
+
+app.use((err, req, res, next) => {
+    if(err.status && err.msg){
+        res.status(err.status).send({msg: err.msg})
+    }
+    else next(err)
+})
```

Comment:
I don't think the app needs this chunk of code yet 👀  Your controller doesn't `catch` an error and put it to the `next` piece of middleware. For the tickets you have coming up, try to only include code that is needed for that particular feature 😄